py/visdom/__init__.py

```diff
@@ -6,6 +6,8 @@
 # This source code is licensed under the license found in the
```

Comment:
As an aside, it's not the best that the entirety of visdom lives in the `__init__.py` file too. For a future refactor 😉

Reply:
Yeah, that's true; the python-client code grew out of control. ^^
Considering some of the issues, there are some things to fix there as well. :angel:

Reply:
For things like this, I'd argue that visdom has reached a scale that we need to hit on more DevX and codebase quality. Before jumping into the small changes, reorganizing would come first. Then I'd probably want to start using `typing`, though that can happen in parallel with small changes (every time you take a pass through to fix a bug, doing code quality improvements in the same pass). Typing and testing make sure that as you're making changes, the code is to some degree keeping track of things by itself so you don't have to.